hack/common.sh

```diff
@@ -17,6 +17,16 @@ error-exit() {
     exit 1
 }
 
+# Extract major version from WMCO_VERSION and map to ocp version
+get_OCP_version(){
```

Comment:
Space after (). I suggest you use this in the community generate.sh. Though with generate.sh I don't understand why you need to infer the version given you will be running the script from the community branch.

Reply:
We agreed that a generic function would be added in common.sh that could be used by both the community script and the release automation script.

See thread: https://redhat-internal.slack.com/archives/CM4ERHBJS/p1682619865397169?thread_ts=1681829477.578399&cid=CM4ERHBJS

Reply:
In that same thread we have discussed inferring the OCP version from the branch. Are we ever going to run the generate.sh script from master?

Reply:
Community branch tracks the master branch